01MIT_INST-MIT/html/homepage/homepage_en.html

```diff
@@ -11,12 +11,12 @@
             </md-card-title>
             <md-card-content>
                 <p>
-                    Primo provides one search for books and e-books, journals and articles, databases, media and more, available in the library and online.</p>
+                    Search Our Collections provides one search for books and e-books, journals and articles, databases, media and more, available in the library and online. Learn more <a href="https://libguides.mit.edu/search-our-collections">about Search Our Collections</a>.</p>
```

Comment:
Hi Adam, Melissa and I just decided to make one more change in this description text - can you make this update before you commit the code? Can you make the whole last sentence in this section be the link, instead of just the segment "about Search Our Collections"?

<a href="https://libguides.mit.edu/search-our-collections">Learn more about Search Our Collections</a>